dev-libs/hiredict/hiredict-1.3.1.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Minimalistic C client library for the Redict database"
+HOMEPAGE="https://codeberg.org/redict/hiredict"
+SRC_URI="https://codeberg.org/redict/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+S=${WORKDIR}/${PN}
+
+LICENSE="BSD LGPL-3"
+SLOT="0/$(ver_cut 1-2)"
```

Comment:
Why do we use `0/1.3` here? The `SONAME` version is set to `1.3.1` and `libhiredict.so.1` symlinks `libhiredict.so.1.3.1`. I don't say it is wrong, I just want to know the reasoning behind.

Reply:
hiredis used to change the soname every minor release, but apparently not at patches

unsure how we'll handle this in hiredict, iirc the idea is to not change the soname unless abi breaks

Reply:
I think we can stick with it for now and we will se how hiredict will handle that.